specification/maps/data-plane/Geolocation/preview/1.0/geolocation.json

```diff
@@ -102,6 +102,7 @@
       "readOnly": true,
       "properties": {
         "countryRegion": {
+          "x-ms-client-flatten": true,
           "$ref": "#/definitions/CountryRegion"
```

Comment:
I would hesitate to recommend client flattening. Presumably, the reason that CountryCode is an object (as opposed to just a string) is to make sure that the API can expand in the future. At which point it can become problematic to "hoist" the properties to the parent.

Reply:
As discussed with @daniel-rocha, the data backend of this API is owned by us, so we could possibly expand its functionalities in the future, which makes @johanste's argument valid. In other APIs where our providers' responses usually do not change, we could consider using flattening. 

@tjprescott I will revert this change if you also agreed with this. Thanks :)

Reply:
Yes, I agree. Let's not use `x-ms-client-flatten` on any response types. If that causes usability issues, we can address in the convenience layer.